src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -389,6 +390,43 @@ public void Dispose()
         /// </summary>
         public void Stop() => _jvmObject.Invoke("stop");
 
+        /// <summary>
+        /// Get the <see cref="VersionSensor.VersionInfo"/> for the Microsoft.Spark assembly
+        /// running on the Spark Driver and make a "best effort" attempt in determining the version
+        /// of Microsoft.Spark.Worker assembly on the Spark Executors.
+        /// </summary>
+        /// <param name="numPartitions">Number of partitions</param>
+        /// <returns>
+        /// A <see cref="DataFrame"/> containing the <see cref="VersionSensor.VersionInfo"/>
+        /// </returns>
+        public DataFrame Version(int numPartitions = 10)
+        {
+            StructType schema = VersionSensor.VersionInfo.s_schema;
+
+            DataFrame sparkInfoDf =
+                CreateDataFrame(
+                    new GenericRow[] { VersionSensor.MicrosoftSparkVersion().ToGenericRow() },
+                    schema);
+
+            Func<Column, Column> workerInfoUdf =
+                Functions.Udf<int>(
+                    i => VersionSensor.MicrosoftSparkWorkerVersion().ToGenericRow(),
+                    schema);
```

Comment:
```suggestion
                    i => VersionSensor.MicrosoftSparkWorkerVersion().ToGenericRow(), schema);
```

Reply:
This suggestion against the example in [csharp-coding-style](https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md)